ocs_ci/deployment/deployment.py

```diff
@@ -491,6 +491,17 @@ def deploy_ocs_via_operator(self, image=None):
 
         cluster_data["metadata"]["name"] = config.ENV_DATA["storage_cluster_name"]
 
+        # Flexible scaling is available from version 4.7
+        if config.ENV_DATA.get("enable_flexible_scaling") and config.DEPLOYMENT.get(
+            "local_storage"
+        ):
+            ocs_version = float(config.ENV_DATA["ocs_version"])
+            if ocs_version < 4.7:
+                error_message = "Flexible scaling can be enabled only on OCS >= 4.7!"
+                logger.error(error_message)
+                raise UnsupportedFeatureError(error_message)
+            cluster_data["spec"]["flexibleScaling"] = True
+
```

Comment:
The storagecluster.yaml will also need to be modified to set the count =3 and replica=1

Reply:
@nbalacha I commented this out above:  https://github.com/red-hat-storage/ocs-ci/pull/3826/files#r572789079 so I am resolving this comment as Oded will handle this in my older comment. Thanks